src/AVFoundation/AVAudioPlayer.cs

```diff
@@ -23,67 +23,57 @@
 //
 using Foundation;
 using ObjCRuntime;
-using AudioToolbox;
 using System;
+using System.Runtime.InteropServices;
 
 #nullable enable
 
 namespace AVFoundation {
 #if !WATCH
 	public partial class AVAudioPlayer {
 
+		[DllImport (Constants.ObjectiveCLibrary, EntryPoint = "objc_msgSend")]
+		unsafe static extern IntPtr objc_msgSend (IntPtr receiver, IntPtr selector, IntPtr arg1, IntPtr arg2);
```

Comment:
I believe the last argument should be a pointer:

```suggestion
		unsafe static extern IntPtr objc_msgSend (IntPtr receiver, IntPtr selector, IntPtr arg1, IntPtr* arg2);
```